src/__experimental__/components/tile/tile.component.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import tagComponent from '../../../utils/helpers/tags';
+import OptionsHelper from '../../../utils/helpers/options-helper';
+import { StyledTile, TileContent } from './tile.style.js';
+
+const Tile = (props) => {
```

Comment:
You could destructure the props in as they're passed into the component but I'm probably being nit-picky so feel free to ignore me
```
const Tile = ({
 as,
 children,
 padding,
 orientation,
 width,
 ...props
}) => {
```

Reply:
Ah yeah, I did do it that way originally, but wasn't sure if we needed to be passing _all_ of the props to tagComponent (on line 38) - moved the destructuring in so that I could pass the complete props object there, but can change it if that's not actually required.

Reply:
It's not that big a deal but if you pass in the `...props` to tagComponent it would still work as it just looks for the `data-` based props. But either way works so I'm happy to approve it as is 👍